packages/worker/src/api/routes/global/tests/auth.spec.ts

```diff
@@ -40,36 +40,43 @@ describe("/api/global/auth", () => {
 
     describe("POST /api/global/auth/:tenantId/reset", () => {
       it("should generate password reset email", async () => {
-        const { res, code } = await config.api.auth.requestPasswordReset(
-          sendMailMock
-        )
-        const user = await config.getUser("test@test.com")
+        await tenancy.doInTenant(config.tenant1User!.tenantId, async () => {
+          const userEmail = `${utils.newid()}@test.com`
```

Comment:
Can use `structures.email()` which will also use a new uuid in the first half of the address